build/scripts/Commandline.fsx

```diff
@@ -77,6 +77,12 @@ module Commandline =
 
         | ["test"; testFilter] -> setBuildParam "testfilter" testFilter
 
+        | ["benchmark"; "non-interactive"; IsUrl elasticsearch; username; password] ->
```

Comment:
I think `"non-interactive"` should be the last argument, to align with the conventions of the other patterns.